src/main/java/Game.java

```diff
@@ -31,16 +29,16 @@ public void run(){
     }
 
     private void init(){
-        gameField = new GameField(new Point(100, 100));
-        gameField.addObject(new GameObject(new Point(10, 10), new Point(10, 10)));
+        this.gameField = new GameField(new Point(100, 100));
+        this.gameField.addObject(new GameObject(new Point(10, 10), new Point(10, 10)));
```

Comment:
"this." is used only during class field init. 

 gameField.addObject(new GameObject(new Point(10, 10), new Point(10, 10)));